doc/how-to/install.md

```diff
@@ -1,6 +1,76 @@
 (howto-install)=
 # How to install MicroCloud
 
+(pre-deployment-requirements)=
+## Pre-deployment requirements
+
+````{tabs}
+```{group-tab} General
+
+The requirements in this section apply to all MicroCloud deployments.
+
+A physical or virtual machine intended for use as a MicroCloud cluster member must meet the following prerequisites:
+
+- Software: 
+  - Ubuntu 22.04 or newer (LTS version recommended)
+  - snapd 2.59 or newer
+
+- Networking:
+  - Fixed IP addresses (DHCP not supported)
+  - Two network interfaces per cluster member, one for intra-cluster communication and one for external connectivity to the uplink network
+    - Partially or fully aggregated networking setups require more interfaces; see: {ref}`howto-ceph-networking`
```

Comment:
`fully disaggregated`?

Reply:
just making sure you're paying attention :)
fixed!